content/docs/practices/the_zen.md

```diff
@@ -0,0 +1,118 @@
+---
+title: The Zen of Prometheus
+sort_rank: 9
+---
+
+# The Zen of Prometheus
+
+**The Zen of Prometheus** is beginner friendly set of core values and guidelines for instrumenting your applications and writing idiomatic alerts using Prometheus.
+This is a document that's intended to maintained by the Prometheus community. Feel free to [contribute](https://github.com/prometheus/docs).
+
+## Instrument first, ask questions later
+
+During development you will never know what questions you need to ask later. Software needs good instrumentation, it's not optional. Metrics are cheap. Use them generously.
```

Comment:
A metric is cheap. Metrics are not cheap.

Metrics without labels are cheap, metrics with labels tend to blow up in cardinality. Making it seem that metrics are cheap would likely lead to more users falling into this trap.

Reply:
Yeah, would be good to be more nuanced here and perhaps link to the cardinality section.